.circleci/config.yml

```diff
@@ -0,0 +1,79 @@
+# Updating CircleCI configuration from v1 to v2
+version: 2
+jobs:
+  build:
+    machine:
+      enabled: true
+    steps:
+      - checkout
+      - run:
+          command: ls
+
+  test:
+    machine:
+      enabled: true
+    steps:
+      - checkout
+      - run:
+          command: docker build -t jupyterhub/jupyterhub .
+      - run:
+          command: docker build -t jupyterhub/jupyterhub-onbuild:${CIRCLE_TAG:-latest} onbuild
+
+  deploy:
+    machine:
+      enabled: true
+    steps:
+      - checkout
+      - run:
+          command: docker login -u $DOCKER_USER -p $DOCKER_PASS -e unused@example.com
+      - run:
+          command: docker push jupyterhub/jupyterhub-onbuild
+
+  release:
+    machine:
+      enabled: true
+    steps:
+      - checkout
+      - run:
+          command: docker login -u $DOCKER_USER -p $DOCKER_PASS -e unused@example.com
+      - run:
+          command: docker push jupyterhub/jupyterhub-onbuild:$CIRCLE_TAG
+
+
+  workflows:
```

Comment:
workflows should be dedented so it aligns with 'jobs' rather than as an item under 'jobs'